### DISCUSSION TITLE
Hide notification when user reads message

### DISCUSSION THREAD

**Discussion by kirasok** [d99kris/nchat#230] [Ideas]
nchat should hide (cancel) notification when user reads message associated with this notification

I think it can be done with `notify-send`:

```shell
notify-send 'nchat' '%1: %2' -p # prints id of notification into STDOUT
notify-send -r $id -t 1 # set timeout for $id notification to 1ms
```

**Comment by d99kris** [comment]
Hi @kirasok - thanks for suggesting a feature and providing some input on how to go about it. I will try to look into this.

If anyones across this thread and know how to do the same in macOS, please feel free to share.
